ci-operator/jobs/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-release-4.6-presubmits.yaml

```diff
@@ -484,30 +484,23 @@ presubmits:
     decoration_config:
       skip_cloning: true
     labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-cluster-image-registry-operator-release-4.6-e2e-openstack
     optional: true
     rerun_command: /test e2e-openstack
+    run_if_changed: ^(.*swift.*|vendor/github\.com/gophercloud/.*)\.go$
```

Comment:
I understand you copied this from the job running on master, but we should also probably include `.*openstack*.` in that regex:
```
❯ find . -name *openstack*
./vendor/github.com/openshift/installer/pkg/types/openstack
./vendor/github.com/gophercloud/utils/openstack
./vendor/github.com/gophercloud/gophercloud/openstack
./manifests/01-registry-credentials-request-openstack.yaml
```